src/modules/wallet/components/row/wallet.css

```diff
@@ -6,3 +6,15 @@
 .balanceShare {
   white-space: nowrap;
 }
+
+.accountSummary {
```

Comment:
you renamed all places account to wallet and here why the usage of account?

Reply:
Fixed